packages/obonode/obojobo-chunks-iframe/viewer-component.js

```diff
@@ -38,7 +38,9 @@ export default class IFrame extends React.Component {
 		this.boundSkipToBottom = this.onClickSkipToBottom.bind(this)
 		this.boundSkipToTop = this.onClickSkipToTop.bind(this)
 
-		MediaUtil.setDefaultZoom(this.props.model.get('id'), this.props.model.modelState.initialZoom)
+		if (!props.readOnly) {
```

Comment:
What's the need for this addition?

Reply:
Because `MediaUtil` is not initialized in the editor and we don't need options for zoom. Not really sure if that is a good solution.